Add a Community page

astorije opened the pull request:
**TODO:**

- [ ] Backers snippet goes out of the screen
- [ ] Youtube embed goes out of the page

MaxLeiter commented on community.md:
```diff
@@ -0,0 +1,74 @@
+
+## IRC channels
+
+If you need help, or want to help others by answering questions they might have, `#thelounge` is our official IRC channel on [Freenode](https://freenode.net/) (`chat.freenode.net:6697` on a secure connection).
```
The channel is for only registered users, so I don’t think it’s worth linking demo here until/unless we change that

xPaw replied:
Demo hostmask is whitelisted for join/chat.

astorije replied:
Ah good catch re: whitelist, I didn't remember that, that's why I did not include a link to demo. Will do!

astorije replied:
Done, what do y'all think?

<img width="723" alt="screen shot 2018-11-04 at 16 55 43" src="https://user-images.githubusercontent.com/113730/47970385-a1207880-e052-11e8-9372-ae316efae8b8.png">

astorije commented on community.md:
```diff
@@ -0,0 +1,77 @@
+If you need help, or want to help others by answering questions they might have, `#thelounge` is our official IRC channel on [Freenode](https://freenode.net/) (`chat.freenode.net:6697` on a secure connection).
+
+{:.center}
+[Click here to join it using our demo](https://demo.thelounge.chat/?join=thelounge).
```
```suggestion
[Click here to join it using our demo](https://demo.thelounge.chat/?join=thelounge)
```

astorije commented:
@xPaw, should be all set for now. I "fixed" the responsiveness, and I moved the sticker form, removed the "you need to be a contributor" caveat as I have sent stickers to whoever wanted them so far :)
